### PR TITLE
Feat/core data

### PR DESCRIPTION
**Pull request by johnpitts**
# Description
Created Core Data model with relationships between Group, User & Post classes

## Checklist

Remove any items which are not applicable.

-  I have performed a self-review of my own code
-  I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by johnpitts** [comment]
> Describe change here
Created Core Data model with relationships between Group, User & Post classes

**Comment by johnpitts** [comment]
Hector Villasano will review code.

**Comment by hectorsvill** [comment]
Everything looks good. Core Data model might need an update after grabbing test data.

File: AllegiancePractise/AllegiancePractise/MODELS/User.swift [hunk 1]
```diff
@@ -6,14 +6,14 @@
...
-    var id: Int               // relationship to GroupUsers userID
-    var username: String
-    var email: String  // varchar
-    var password: String
-    var location: Int // zipcode
-    var image: String    // to convert to UIImage / data
-}
+//import Foundation
+//
+//class User: Codable {
```
Comment (hectorsvill) [inline AllegiancePractise/AllegiancePractise/MODELS/User.swift:11]: User Class looks great!

Comment (johnpitts) [reply AllegiancePractise/AllegiancePractise/MODELS/User.swift:11]: right-on bro-heem

**Pull request merged**